### DISCUSSION TITLE
电脑在没有物理屏幕，使用虚拟屏幕，使用远程控制时，maa无法正常显示

### DISCUSSION THREAD

**Discussion by Huayu13154E5** [MaaAssistantArknights/MaaAssistantArknights#4108] [General]
![QQ截图20230329183415](https://user-images.githubusercontent.com/65339022/228507556-eff4a423-dd24-4b9d-b555-8a1ad3c7b634.png)

**Comment by MistEO** [comment]
这种工况，试试用 python 接口集成（

**Reply by Huayu13154E5** [reply]
大佬，咱没懂😳

**Reply by Huayu13154E5** [reply]
懂了 我去试试
